infra/gcp/terraform/k8s-infra-prow-build-trusted/secrets.tf

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 locals {
   build_cluster_secrets = {
     capdo-quayio-registry-secret = {
-      group = "sig-cluster-lifecycle"
+      group  = "sig-cluster-lifecycle"
       owners = "k8s-infra-staging-cluster-api-do@kubernetes.io"
     }
     cncf-ci-github-token = {
       group  = "sig-testing"
       owners = "k8s-infra-ii-coop@kubernetes.io"
     }
+    eks-prow-build-cluster-kubeconfig = {
+      group  = "sig-k8s-infra"
+      owners = "k8s-infra-aws-admins@kubernetes.io"
```

Comment:
I don't know if this is right, let me know if you have better values for this.

Reply:
Should be good.